web/js/panel.js

```diff
@@ -2574,7 +2586,7 @@ $(document).ready(function() {
     }
     // setup the functional menu items
     $("#navbar-panel-reload > a").attr("href", location.href);
-    $("#navbar-panel-xml > a").attr("href", location.href + "?format=xml");
+    $("#navbar-panel-xml > a").attr("href", location.pathname + "?format=xml");
```

Comment:
What's the difference between href and pathname? I'm asking so that I can learn something new.

Reply:
href includes existing parameters, pathname does not. 
so this change avoids "`?protect=xml?format=xml`" which is not valid.